examples/guides/add-legends/step-5.html

```diff
@@ -46,12 +46,12 @@ <h1 id="light">Rail Accidents by Weather</h1>
 
         const source = new carto.source.Dataset("railroad_accidents");
         const viz = new carto.Viz(`
-            @myRamp: ramp($weather,[darkorange,darkviolet,darkturquoise])
+            @myRamp: ramp($weather, [darkorange, darkviolet, darkturquoise])
```

Comment:
I would use here a variable for the palette and then the ramp and opacity in the color expression

Reply:
Yeah, that sounds as a better approach